perma_web/perma/templates/docs/faq.html

```diff
@@ -41,120 +42,153 @@ <h1>Perma.cc user guide <span class="subsection">FAQ</span></h1>
     <div class="col-sm-8 reading-body">
       <h2 class="body-ah" id="general">General</h2>
 
-      <h3 class="body-bh" data-href="who-is">Who runs Perma.cc?</h3>
```

Comment:
Thanks for tidying these no-longer-used `data-href` attributes. In the few cases where it's an `id`, not a `data-href`, it might be worth keeping the id and/or checking to see if there are any links around to that element? When an element has an `id`, you can link straight to that spot on the page, e.g. `http://perma.cc/docs/faq#librarian`. I don't know if we do, though, or if that's just more outdated markup.....